root/application/src/main/java/at/fhv/itb5c/application/ApplicationFacade.java

```diff
@@ -353,4 +401,84 @@ public MatchDTO saveMatch(String sessionId, MatchDTO matchDTO, DepartmentDTO dep
 		}
 		return null;
 	}
+
+	public Boolean rsvp(String sessionId, TeamDTO team, TeamInvitationStatus answer) {
+		if (team == null || answer == null) {
+			return null;
+		}
+		if (!answer.equals(TeamInvitationStatus.Accepted) && !answer.equals(TeamInvitationStatus.Declined)) {
+			log.warn("rsvp: TeamInvitationStatus must be set to Accepted or Declined!");
+			return null;
+		}
+		try {
+			UserDTO player = getCurrentUser(sessionId);
+			Team teamEntity = ConverterTeamDTO.toEntity(team);
+			teamEntity.setMemberStatus(player.getId(), answer);
+			return PersistenceFacade.getInstance().saveOrUpdate(teamEntity) != null;
+		} catch (Exception e) {
+			log.error(e.getMessage());
+			return false;
+		}
+	}
+
+	public MessageDTO getMessage(String sessionId) {
+		QueueManager qm = new QueueManager(SessionManager.getInstance().getUserId(sessionId).toString());
```

Comment:
Well, a QueueManager works only for a specific queue; so, we could save a list of all existing queue managers somewhere if performance would be a problem, but I don't think we really need that right now. (Probably, this will also allow us to use a Stateless Session Bean instead)